### DISCUSSION TITLE
Create a board for highly requested ideas

### DISCUSSION THREAD

**Discussion by peepo5** [tenacityteam/tenacity-legacy#654] [Ideas]
There are a huge amount of ideas brought by a lot of people from many communities, however without much incentive it can be hard to make progress in this manner.

My idea is to make a board with highly sought-after additions where people can see what is highly requested / needed so they can contribute. On top of this idea is using bounties. Funding could be used (from donations) to give a reward for people that create PR's that add these needed features, which would be a really good method of increasing progress. The project [invidious](https://github.com/iv-org/invidious) have done this and it has been a good way of stirring incentive.

Just as a starter, here are some ideas.
- Replacement of wxwidgets or replace GUI with an alternative like qt or gtk
- Make waveform creation smooth (60fps+) while recording
- UI modernization / overhaul
- Files / clip window
- Configure waveform colors

etc

**Comment by Semisol** [comment]
> On top of this idea is using bounties. Funding could be used (from donations) to give a reward for people that create PR's that add these needed features, which would be a really good method of increasing progress.

I understand that, but currently I do not think we should give out our money as we do not know what kind of expenses we might use, and I would prefer to keep a buffer & not spend money on unnecessary things.

Otherwise, instead of a list would a label on [`enhancement`](https://github.com/tenacityteam/tenacity/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aenhancement) issues work?

**Reply by peepo5** [reply]
I think a list would work better.
Having an organized list means its more checklist-like and you can sort by priority etc.
I think it would be useful because you could also discuss additions that would be much needed.

**Reply by Semisol** [reply]
It could be a tracking issue for 1.0 (+ RC) I assume?
Otherwise, I don't know how the list would be hosted.

**Reply by peepo5** [reply]
@Semisol 
I don't understand what you mean.
It could just be a thread in the discussions tab

**Reply by poyribn** [reply]
@peepopoggers the Tenacity project wants to be independent from GitHub,
so it discourages using GitHub-specific features (like the discussions tab), which could make it depend on GitHub.

mailing lists are an alternative that does not rely on GitHub.
and when the project becomes mature enough, they might just disable the discussions tab altogether in favor of using just the mailing lists.

**Reply by peepo5** [reply]
You could use cryptpad.fr but im not sure how suggestions could be made

**Reply by Semisol** [reply]
> im not sure how suggestions could be made

Mailing lists

> You could use cryptpad.fr

Could try that one, thanks
